src/templates/combat/combatant.hbs

```diff
@@ -0,0 +1,50 @@
+<li class="combatant actor directory-item flexrow {{this.css}}" data-combatant-id="{{this.id}}">
+    <img class="token-image" data-src="{{this.img}}" alt="{{this.name}}" />
+    <div class="token-name flexcol">
+        <h4>{{this.name}}</h4>
+        <div class="combatant-controls flexrow">
+            {{#if ../user.isGM}}
+            <a class="combatant-control {{#if this.hidden}}active{{/if}}" aria-label="{{localize 'COMBAT.ToggleVis'}}"
+                role="button" data-tooltip="COMBAT.ToggleVis" data-control="toggleHidden">
+                <i class="fas fa-eye-slash"></i>
+            </a>
+            <a class="combatant-control {{#if this.defeated}}active{{/if}}"
+                aria-label="{{localize 'COMBAT.ToggleDead'}}" role="button" data-tooltip="COMBAT.ToggleDead"
+                data-control="toggleDefeated">
+                <i class="fas fa-skull"></i>
+            </a>
+            {{/if}}
+            {{#if this.canPing}}
+            <a class="combatant-control" aria-label="{{localize 'COMBAT.PingCombatant'}}" role="button"
+                data-tooltip="COMBAT.PingCombatant" data-control="pingCombatant">
+                <i class="fa-solid fa-bullseye-arrow"></i>
+            </a>
+            {{/if}}
+            {{#unless ../user.isGM}}
+            <a class="combatant-control" aria-label="{{localize 'COMBAT.PanToCombatant'}}" role="button"
+                data-tooltip="COMBAT.PanToCombatant" data-control="panToCombatant">
+                <i class="fa-solid fa-arrows-to-eye"></i>
+            </a>
+            {{/unless}}
+            <div class="token-effects">
+                {{#each this.effects}}
+                <img class="token-effect" src="{{this}}" />
+                {{/each}}
+            </div>
+        </div>
+    </div>
+
+    {{#if this.hasResource}}
+    <div class="token-resource">
+        <span class="resource">{{this.resource}}</span>
+    </div>
+    {{/if}}
+    <div class="cosmere-combatant-buttons">
+        <a class="cosmere-turn-speed-control"></a>
+            {{#if this.activated}}
+            <a class="cosmere-activate-control-activated"></a>
+            {{else}}
+            <a class="cosmere-activate-control"></a>
+            {{/if}}
+    </div>
```

Comment:
Having tried it, it would be nice to have some tooltips (using `data-tooltip` attr) that indicate what the buttons do